src/components/Hero.astro

```diff
@@ -0,0 +1,24 @@
+---
+import Button from './Button.astro';
+import { Image } from 'astro:assets';
+
+const { headline, subhead, buttonsData = [], heroImage, bottomMargin = '' } = Astro.props;
+
+const containerClasses = `container flex flex-col flex-wrap mx-auto px-[0.5rem] sm:px-[1rem] md:px-[1.5rem] md:flex-row md:px-[2rem] xl:px-[1.5rem] 2xl:px-[3.62rem] 3xl:px-[6.62rem] ${bottomMargin}`;
+---
+<div class={containerClasses}>
+  <div class="w-12/12 md:order-last md:w-1/2">
+      <Image class="w-[30rem] mx-auto md:w-full" src={heroImage} alt={headline} densities={[1, 2, 3, 4]} />
+  </div>
+  <div class="w-12/12 mb-[1.5rem] mt-[0.78375rem] md:my-auto md:pr-[2.02625rem] md:w-1/2">
+    <h1 class="mb-[1rem] text-[2.5rem] text-black 2xl:text-[3.75rem]">{headline}</h1>
```

Comment:
@Mat-Mwi can you please adjust the font sizes for breakpoint md as noted here: https://www.figma.com/design/q8NAB0WClWHGTbajxKeskD/Sentry---Fair.io-(8)?node-id=1015-1377&node-type=frame&m=dev